src/content/docs/distributed-tracing/infinite-tracing/set-trace-observer.mdx

```diff
@@ -153,16 +295,70 @@ To set up a trace observer:
           </Callout>
      </Collapser>
    </CollapserGroup>
-6. (Optional) There are several ways to [configure Infinite Tracing](/docs/distributed-tracing/infinite-tracing). This configuration can wait until after you've completed the enable procedures.
-7. This procedure is complete. Next, return to finish any remaining instructions for the tracing tool you started enabling:
 
-   * [Language agents](/docs/understand-dependencies/distributed-tracing/enable-configure/language-agents-enable-distributed-tracing#configure-agents)
-   * [Third-party telemetry integrations](/docs/understand-dependencies/distributed-tracing/enable-configure/integrations-enable-distributed-tracing#configure-integration) (OpenTelemetry and others)
-   * [Trace API](/docs/understand-dependencies/distributed-tracing/trace-api): once the trace observer is set up, you're finished and can start instrumenting your application.
+### Initial trace observer configuration for browser, mobile, and Lambda [#initial-config]
+
+Once you have a trace observer, you need to specify which data sources (services, apps, or entities) should have their data sent to the Infinite Tracing trace observer. Before you select your data sources, familiarize yourself with these issues:
```

Comment:
I thought "issues" seemed a little too close to "problems". What do you think of this? Does it feel awkward?

```suggestion
Once you have a trace observer, you need to specify which data sources (services, apps, or entities) should have their data sent to the Infinite Tracing trace observer. Before you select your data sources, familiarize yourself with the following considerations:
```